src/node_modules/graph-view/node-ui.js

```diff
@@ -24,33 +28,40 @@ var style = {
 
 module.exports = React.createClass({
   render: function () {
-
     var d = this.props.datum
-    var rect = this.props.svgRect
 
     var w = 40
     var h = 40
     var offset = w / 2
 
-    var x = d.x + rect.left - offset
-    var y = d.y + rect.top - offset
+    var x = d.x - offset
+    var y = d.y - offset
 
     return (
-      r(Link, {
-        href: router.format({ id: d.id }),
-      }, [
-        r.img({
-          src: d.image,
-          height: h,
-          width: w,
-          style: extend(
-            style.node,
-            { top: y, left: x },
-            style.circleClip20
-          ),
-          className: 'node circle-clip-20',
-        })
-      ])
+      React.createElement('image', {
+        'xlink:href': d.image,
+        height: String(h),
+        width: String(w),
+        x: String(x),
+        y: String(y),
+        clipPath: 'url(#circle-clip-20)',
+        className: 'node circle-clip-20',
+        style: style.circleClip20,
+        onClick: this.handleClick
+      })
     )
   },
+
+  href: function () {
+    var d = this.props.datum 
+    debug('d', d)
+    return router.format({ id: this.props.datum.id })
+  },
+
```

Comment:
@ahdinosaur using a onclick handler to router directly as there were problems wrapping an ```<image>``` with a ```Link``` (even after my attempt at hacking on ```Link```)

getting an error ![](http://i.imgur.com/47QitsL.png)